pkg/utils/charts.go

```diff
@@ -29,7 +29,10 @@ func PathExists(path string) bool {
 
 var chartFolder = ""
 
-// Get the directory of charts
+/**
+*
+* Get the directory of charts
```

Comment:
Please follow the godoc [convention](https://blog.golang.org/godoc) on comments. That is, using func name as the inital word. Thanks.

Reply:
are you sure?other classmate did not use the func name.

Reply:
![image](https://user-images.githubusercontent.com/19778214/102004615-6323d180-3d4d-11eb-9ef9-800dccf3edca.png)

Reply:
Yeah, I'm sure.  You are suggested to do something like this:
```suggestion
// GetChartsDirectory gets the directory of the charts
```